tools/dataset_converters/yolo2coco.py

```diff
@@ -0,0 +1,287 @@
+"""This script helps to convert yolo-style dataset to the coco format.
+
+Usage:
+    $ python yolo2coco.py --image-dir /path/to/dataset   # the root dir
+
+Note:
+    1. Before running this script, please make sure the root directory
+    of your dataset is formatted in the following struction:
+    .
+    └── $ROOT_PATH
+        ├── classes.txt
+        ├── labels
+        │    ├── a.txt
+        │    ├── b.txt
+        │    └── ...
+        ├── images
+        │    ├── a.jpg
+        │    ├── b.png
+        │    └── ...
+        └── ...
+    2. The script will automatically check whether the corresponding
+    `train.txt`, ` val.txt`, and `test.txt` exist under `image-dir` or not.
+    If these files are detected, the script will organize the dataset.
+    The image paths in these files must be ABSOLUTE paths.
+    3. Once the script finishes, the result files will be saved in the
+    directory named 'annotations' in the root directory of your dataset. The
+    corresponding images will be kept in `.jpg` format. The 'annotations'
+    folder may look like this in the root directory:
+    .
+    └── $ROOT_PATH
```

Comment:
└── $ROOT_PATH
        ├── annotations
        │    ├── train.json
        ├── classes.txt
        ├── labels
        │    ├── a.txt
        │    ├── b.txt
        │    └── ...
        ├── images
        │    ├── a.jpg
        │    ├── b.png
        │    └── ...
        └── ...

Reply:
Add a sentence at the end of docstr: After converting to coco, you can use the  `tools/analysis_tools/browse_coco_json.py` script to visualize whether it is correct